wildvvad/utils/downloadUtils.py

```diff
@@ -0,0 +1,86 @@
+import os.path
+import shutil
+import urllib.request
+import zipfile
+from tqdm import tqdm
+
+
+class downloadUtils:
+    def __init__(self):
```

Comment:
if you are not offering to change the urls and folder structure these values should be class members